src/database/sqlite/transactions.md

```diff
@@ -0,0 +1,50 @@
+## Using transactions
+
+[![rusqlite-badge]][rusqlite] [![cat-database-badge]][cat-database]
+
+[`Connection::open`] will open the `cats.db` database from the top recipe.
+
+Begin a transaction with [`Connection::transaction`]. Transactions will
+roll back unless committed explicitly with [`Transaction::commit`].
+
+```rust,no_run
+extern crate rusqlite;
+
+use rusqlite::{Connection, Result};
+
+fn main() -> Result<()> {
+    let mut conn = Connection::open("cats.db")?;
+
+    successful_tx(&mut conn)?;
+
+    let res = rolled_back_tx(&mut conn);
+    assert!(res.is_err());
+
+    Ok(())
+}
+
+fn successful_tx(conn: &mut Connection) -> Result<()> {
+    let tx = conn.transaction()?;
+
+    tx.execute("delete from cat_colors", &[])?;
+    tx.execute("insert into cat_colors (name) values (?1)", &[&"lavender"])?;
+    tx.execute("insert into cat_colors (name) values (?1)", &[&"blue"])?;
+
+    tx.commit()
+}
+
+fn rolled_back_tx(conn: &mut Connection) -> Result<()> {
+    let tx = conn.transaction()?;
+
+    tx.execute("delete from cat_colors", &[])?;
+    tx.execute("insert into cat_colors (name) values (?1)", &[&"lavender"])?;
+    tx.execute("insert into cat_colors (name) values (?1)", &[&"blue"])?;
+    // duplicate color
```

Comment:
Instead of having a comment in the code, will you add to the description that when a duplicate color attempts to insert into a unique column, the transaction rolls back?